content/blog/2022-03-31-training-and-saving-models-on-a-dedicated-runner.md

```diff
@@ -0,0 +1,284 @@
+---
+title:
+  Training and saving models with CML on a dedicated AWS EC2 runner (part 1)
+date: 2022-03-31
+description:
+  In this guide we will show how you can use CML to automatically retrain a
+  model and save its outputs to your Github repository using a provisioned AWS
+  EC2 runner.
+descriptionLong: |
+  We can use CML to automatically retrain models whenever data, model code,
+  or parameters change. In this guide we show how to create a pipeline that
+  provisions an AWS EC2 instance to retrain a model and save the output on
+  a regular basis. This way we can prevent drift by ensuring that our model
+  always uses the latest input data.
+picture: 2015-05-01/post-image.jpeg
```

Comment:
Image needs to be added.  Create folder to go here: dvc.org/static/uploads/images/2022-04-07